.appveyor.yml

```diff
@@ -94,7 +94,8 @@ install:
   - ps: $env:PYTABLES_VERSION=(sls "tables==(.*)" .\etc\requirements.txt -ca).matches.groups[1].value
   - ps: $env:H5PY_VERSION=(sls "h5py==(.*)" .\etc\requirements.txt -ca).matches.groups[1].value
   - ps: $env:TALIB_VERSION=(sls "TA-Lib==(.*)" .\etc\requirements_talib.txt -ca).matches.groups[1].value
-  - conda create -n testenv --yes -q --use-local pip python=%PYTHON_VERSION% numpy=%NUMPY_VERSION% pandas=%PANDAS_VERSION% scipy=%SCIPY_VERSION% ta-lib=%TALIB_VERSION% bcolz=%BCOLZ_VERSION% numexpr=%NUMEXPR_VERSION% pytables=%PYTABLES_VERSION% h5py=%H5PY_VERSION% -c quantopian -c https://conda.anaconda.org/quantopian/label/ci
+  - ps: $env:CERTIFI_VERSION=(sls "certifi==(.*)" .\etc\requirements_certifi.txt -ca).matches.groups[1].value
```

Comment:
Probably worth adding comments around why we need to install `certifi` like this (since it's different from most of the requirements).